conans/cli/commands/remove.py

```diff
@@ -0,0 +1,10 @@
+from conans.cli.api.conan_api import ConanAPIV2
+from conans.cli.command import conan_command, COMMAND_GROUPS, Extender
+
+
+@conan_command(group=COMMAND_GROUPS['creator'])
+def remove(conan_api: ConanAPIV2, parser, *args, **kwargs):
```

Comment:
Not sure if ``remove`` is creator only, it is not that unusual that consumers want to remove installed packages, clean the cache, etc.